antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -53,22 +53,16 @@
             const data = hits.reduce((data, hit) => {
               const d = {}
               d.url = hit.url
+              var section = hit.hierarchy.lvl1
+              if (hit.hierarchy.lvl0 !== null) section = section + ' [' + hit.hierarchy.lvl0 + ']'
               var breadcrumbs = Object.values(hit.hierarchy)
-                .slice(1)
+                .slice(2)
                 .filter((lvl) => lvl !== null)
                 .join(' &raquo; ')
-              if (breadcrumbs !== '') {
-                d.breadcrumbs = breadcrumbs
-              } else {
-                d.breadcrumbs = hit.hierarchy.lvl0
-              }
-              if (hit._snippetResult !== undefined) {
-                d.snippet = hit._snippetResult.content.value
-              } else {
-                d.snippet = ''
-              }
-
-              const section = hit.hierarchy.lvl0
+              
```

Comment:
There is a trailing space here according to netlify, same case with line 65.  Remove it and then we could test the preview.